src/cljs/athens/core.cljs

```diff
@@ -98,6 +117,7 @@
   (style/init)
   (stylefy/tag "body" style/app-styles)
   (listeners/init)
+  (init-datalog-console)
```

Comment:
I think we should put this under dev/debug flag.

Reply:
Yea I'm 50/50 on whether it should be a debug only option. I kind of like the idea of giving users an admin console into their own database. Only downside I can see is some resource usage overhead and the need for more security as the console becomes more powerful.

Reply:
I think this can be available to everyone.